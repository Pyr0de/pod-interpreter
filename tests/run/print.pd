print "hello";
print 10+10;
print 10-10;
print;
print 10;
