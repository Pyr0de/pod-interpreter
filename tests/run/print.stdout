ExitCode: 0
hello
20
0

10
